internal/tui/list.go

```diff
@@ -239,8 +246,6 @@ func (m *ListModel) Update(msg tea.Msg) (Component, tea.Cmd) {
 		case "n":
 			editModel := NewEditModel(nil)
 			return editModel, nil
-		case "s":
-			return m, func() tea.Msg { return scanMsg{} }
```

Comment:
Why did we remove this key binding?

Reply:
No, manually controlling the scan is a feature I want the user to have, regardless of the automated lifecycle. The user should still be able to stop the active scanning in the list view with the "S" key, and they should be able to force a single scan in the list view with the "s" key.